beavis-ci/beavis-ci.sh

```diff
@@ -107,65 +117,43 @@ else
     ext="nbconvert.ipynb"
     target="rendered"
 fi
-mkdir -p log
-webdir="https://github.com/LSSTDESC/DC2-analysis/tree/${branch}/tutorials"
-
-# Some notebooks have whitespace in their names - fix this first:
-for notebook in *.ipynb; do
-    ipynbfile="$( echo "$notebook" | sed s/' '/'_'/g )"
-    if [ $ipynbfile != "$notebook" ]; then
-        mv -f "$notebook" $ipynbfile
-    fi
-done
-
-# Which notebooks to run? Either all of them, or the ones listed in README.rst:
-if [ $all -eq 0 ]; then
-    notebooks="$( grep '`ipynb' README.rst |& grep -v "No such file" | cut -d"<" -f2 | cut -d">" -f1 | sed s/'%20'/'_'/g | sed s/'%3A'/':'/g )" 
-fi
-# If README.rst is not present or empty of ipynb instances, default to running everything:
-if [ $all -gt 0 ] || [ ${#notebooks[0]} -eq 0 ]; then
-    notebooks="$( ls *.ipynb )"
-fi
 
+notebooks=`find . -path '*/.ipynb_checkpoints/*' -prune -o -name '*.ipynb' -print`
```

Comment:
I guess this is where we abandon parsing the README - I think that's OK given our plan to switch to writing READMEs automatically.